apps/arweave/src/ar_data_sync_worker.erl

```diff
@@ -267,3 +267,10 @@ sync_range({Start, End, Peer, TargetStoreID, RetryCount} = Args) ->
 					end
 			end
 	end.
+
+get_target_packing(StoreID) ->
+	{ok, Config} = application:get_env(arweave, config),
```

Comment:
PR looks good! I'd suggest loading the config once during `init` and storing it in the #state. I'm not sure how significant an impact the get_env call will have (may be minor), but during heavy syncing we might expect this call to happen hundreds of times a second (and billions of times overall) so it may add up

Reply:
The actual `ar_storage_module:get_packing(StoreID)` us O(n) as it traverses the list. Is this will become a problem, this is potential candidate for optimization.